### ISSUE TITLE
Location Sharing and Google+ Crashes

### ISSUE THREAD

**Issue by scento** [microg/android_packages_apps_GmsCore#96]
When one tries to enable location sharing via *Google+*, the app crashes while trying to open an intent to `com.google.android.gms.location.settings.LOCATION_SHARING`. There are some legitimate reasons to support this feature within the user's restrictions and even if we should not support location sharing, `GmsCore` should contain some fallback code, which at least prevents crashing in case one tries to access the location sharing settings.

**Comment by Shished** [comment]
Installing [maps API v1](https://github.com/microg/android_frameworks_mapsv1) fixed this issue for me.

**Comment by vmavromatis** [comment]
I tried installing mapsv1 again (which I thought was part of microg anyway?) but still getting crashes.

**Comment by ale5000-git** [comment]
microG GmsCore contains Maps API v2.
Maps API v1 is completely separate.

**Comment by vmavromatis** [comment]
@ale5000-git yes I flashed the [zip](https://github.com/microg/android_frameworks_mapsv1/releases) but still got crashes when trying to share my location.

If I could find an alternative that lets me share my location forever with someone I would use that btw... I'm not so invested in google maps, it's just that my family uses it
